pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -783,3 +833,72 @@ func (a byPVCSize) Less(i, j int) bool {
 func claimToClaimKey(claim *v1.PersistentVolumeClaim) string {
 	return fmt.Sprintf("%s/%s", claim.Namespace, claim.Name)
 }
+
+// isCSIMigrationOnForPlugin checks if CSI migrartion is enabled for a given plugin.
+func isCSIMigrationOnForPlugin(pluginName string) bool {
+	switch pluginName {
```

Comment:
Since the `features.CSIMigration` enablement check is done in the only caller of `isCSIMigrationOnForPlugin`, it's okay to skip the check for `features.CSIMigration` here. A comment calling this out explicitly would be nice. We will need to have this call into something like [this](https://github.com/kubernetes/kubernetes/pull/83098/files#diff-23d6020bc9cdbf469af5eeebbe2821f6R55) once that PR lands.